core/modules/filters/equals.js

```diff
@@ -0,0 +1,63 @@
+/*\
+title: $:/core/modules/filters/equals.js
+type: application/javascript
+module-type: filteroperator
+
+Filter operator for comparing text for equality
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/*
+Export our filter function
+*/
+exports.equals = function(source,operator,options) {
+	var results = [],
+		fieldname = (operator.suffix || operator.operator || "title").toLowerCase();
+	if(operator.prefix === "!") {
+		if(operator.regexp) {
+			source(function(tiddler,title) {
+
+				var text = title ;
```

Comment:
Are you sure you want to masquerade text as title?

Reply:
Yes. I avoid unnecessary refactoring.